src/foam/u2/view/DraftDetailView.js

```diff
@@ -0,0 +1,69 @@
+/**
+ * @license
+ * Copyright 2019 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.u2.view',
+  name: 'DraftDetailView',
+  extends: 'foam.u2.View',
+  documentation: `
+    A detail view that holds off on updating the given object until the user clicks save.
```

Comment:
I would drop a TODO somewhere in here that we need nested property change event support for this to be truly robust.  Otherwise if you change a property of property of your data, you won't update hasDiff.

Reply:
Good call. Done.